cbins13/dotTheory---FE
# FE-863: Applying Animations & Transitions

## Related Tickets
- FE-863

## Description
- This pull request is created for the module FE-402, where the checklist includes:

1. Add hover, active, and focus states to the cards on your Browse Pages
2. Add hover, active, and focus states to the cards on your Home Page
3. Add hover, active, and focus states to the hyperlinks on your Detail Pages
4. Add hover, active, and focus states to the links on your navigation and footer
5. Create an About Page and use SVG animations to make the page stand out from the rest of the site

## Validation
- Install Sass, and Code Editor of your choice (VSCode, Atom, Sublime)
- Open the index.html on Live Server via VSCode to view the detail page, or open the index.html on a browser
- Open terminal and navigate to the the root directory of the project (html-css-project)
- Enter the command sass --watch scss:css :compressed 
- Under the css folder, open the application.css file to view the output css, or
- Under the browser, press the f12 key to view an element and see the pseudo classes containing animations/transitions
